pulsar.el

```diff
@@ -464,6 +476,40 @@ This is a buffer-local mode.  Also check `pulsar-global-mode'."
 
 (make-obsolete 'pulsar-setup nil "0.3.0")
 
+;; Lifted from Emacs 30 to allow pulsar to remain backward compatbile
+;; with Emacs earlier than Emacs 29.1. TODO: Deprecate this at some
+;; point to prefer Emacs core.
+(defun pulsar--function-alias-p (func &optional _noerror)
+  "Return nil if FUNC is not a function alias.
+If FUNC is a function alias, return the function alias chain."
+  (declare (advertised-calling-convention (func) "30.1")
+           (side-effect-free error-free))
+  (let ((chain nil))
+    (while (and (symbolp func)
+                (setq func (symbol-function func))
+                (symbolp func))
+      (push func chain))
+    (nreverse chain)))
+
+;; This is essentially the inverse of function-alias-p for a list of
+;; function symbols.
+(defun pulsar--find-fn-aliases (fns)
+  "Return a list of aliases for the FNS symbols."
+  (let ((aliases))
+    (mapatoms (lambda (sym)
+                (when (and
+                       (fboundp sym)
```

Comment:
Now that I think about it more, we probably should change ```fboundb``` to ```commandp``` since ```pulsar``` works only with interactive commands.

Reply:
Done.